server/zoom/www/handlers/regex_application_state_handler.py

```diff
@@ -0,0 +1,117 @@
+import httplib
+import json
+import logging
+import os.path
+import re
+
+import tornado.web
+
+from zoom.common.decorators import TimeThis
+
+
+class RegexApplicationStateHandler(tornado.web.RequestHandler):
+    @property
+    def data_store(self):
+        """
+        :rtype: zoom.www.cache.data_store.DataStore
+        """
+        return self.application.data_store
+
+    @property
+    def application_state_cache(self):
+        """
+        :rtype: zoom.www.cache.application_state_cache.ApplicationStateCache
+        """
+        return self.application.data_store.application_state_cache
+
+    @property
+    def app_state_path(self):
+        """
+        :rtype: str
+        """
+        return self.application.configuration.application_state_path
+
+    @TimeThis(__file__)
+    def get(self, path):
+        """
+        @api {get} /api/v2/application/states/[:id] Get Application State
+        @apiVersion 2.0.0
+        @apiName GetAppState
+        @apiGroup ApplicationState
+        @apiSuccessExample {json} Success-Response:
```

Comment:
can you add an `@apiDescription` here describing that you can do glob matching now? Otherwise you would have no idea